### DISCUSSION TITLE
GSOC 2022: Interested in Multi-version Spatial Indexing Project.

### DISCUSSION THREAD

**Discussion by saisuraj27-729** [koordinates/kart#598] [Google Soc / SoD]
Hi sir, I am Sai, and I am currently pursuing B.Tech 2nd year in the Indian Institute of Technology Tirupati. I have a good knowledge of C++, python, and data science. I also did **two not-so-basic but good projects in data science**. Also, I am **a Civil engineer, so I also have a good idea of using QGIS software as I did some projects related to surveying using QGIS** in my last semester. I am very much interested in contributing to the Multi-version Spatial Indexing project through GSOC 2022. I have good knowledge of C++, Python **Data Structures, and Algorithms as I am a 3-star coder on the Code Chef** platform. But I don't have that level of understanding in contributing to big projects. Can you **please give me a brief idea about the ultimate goal of this project and any materials related to it**, if possible? I will go through them thoroughly and get back to you in 2 days. I am **One-hundred-percent interested in working with you on this project in GSOC 2022.** So, it would be very beneficial to me if you could reply as soon as possible.
It is mentioned that @craigds @rcoup are the mentors for this project, So can you please guide me. 
Thank you, Regards.

**Comment by rcoup** [comment]
Hi Sai, 

See the comments on https://github.com/koordinates/kart/discussions/571. That should give you some background and there's some further links to read there. Ask more questions once you've digested that.
